### PR TITLE
Feature/base dictionary

### PR DESCRIPTION
**Pull request by tgorskievo**
Added `BaseDictionary`.
Updated functional tests which cover dictionary area.
Updated documentation.

### CODE REVIEW

File: MIGRATION-2.0.MD [hunk 1]
```diff
@@ -0,0 +1,100 @@
+# Kakunin - automated testing framework
+#### Migration to 2.0 version
+
+What needs to be done:
+1. Remove all locators from Page Objects
+```javascript
+this.exampleLocator = by.css('.unique-class');
+```
+
+needs to be change to:
```
Comment (aherok) [inline MIGRATION-2.0.MD]: typo: changed

File: MIGRATION-2.0.MD [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+2. Remove headers `| element | value |` from all steps
+```gherkin
+And there are "equal 1" following elements for element "rows":
+   | element   | value  |
+   | firstName | t:John |
+   | lastName  | t:Doe  |
+```
+
+needs to be change to:
```
Comment (aherok) [inline MIGRATION-2.0.MD]: the same typo

**Comment by tgorskievo** [comment]
@aherok fixed typos

**Comment by tgorskievo** [comment]
@aherok fixed typos

**Pull request merged**